Changes

```diff
@@ -24,26 +24,26 @@ Working version
   bundles.
   (whitequark)
 
-* GPR#659: Remove support for SPARC native code generation
-  (Mark Shinwell)
+- MPR#7447, GPR#995: incorrect code generation for nested recursive bindings
+  (Leo White and Jeremy Yallop, report by Stephen Dolan)
 
 - MPR#7501, GPR#1089: Consider arrays of length zero as constants
-- PR#7501, GPR#1089: Consider arrays of length zero as constants
+- MPR#7501, GPR#1089: Consider arrays of length zero as constants
```

Comment:
Thanks, fixed. (I think these issues come from a weird git fuzzy merge glitch, at least not from the 4.05 version.). I kept the change in trunk and removed the one in 4.04.1 -- I don't think this change was even available in 4.05.